e2e/azure-resources/azure-container-registry-airgap.bicep

```diff
@@ -0,0 +1,27 @@
+@minLength(5)
+@maxLength(50)
+
+@description('Provide a globally unique name of your Azure Container Registry')
+param acrName string
+
+resource acrResource 'Microsoft.ContainerRegistry/registries@2023-01-01-preview' = {
```

Comment:
any reasons the AVM wasn't used , instead of the raw resources ?

Reply:
https://github.com/Azure/bicep-registry-modules/tree/main/avm/res/container-registry/registry